src/core/accumulators/Correlator.hpp

```diff
@@ -216,16 +223,16 @@ class Correlator : public AccumulatorBase {
 
   int hierarchy_depth; ///< maximum level of data compression
   int m_tau_lin;       ///< number of frames in the linear correlation
-  int m_dim_corr;
-  double m_dt;      ///< time interval at which samples arrive
-  double m_tau_max; ///< maximum time for which the correlation should be
-                    ///< calculated
+  size_t m_dim_corr;   ///< number of columns for the correlation
+  double m_dt;         ///< time interval at which samples arrive
+  double m_tau_max;    ///< maximum time for which the correlation should be
+                       ///< calculated
 
   std::string compressA_name;
   std::string compressB_name;
   std::string corr_operation_name; ///< Name of the correlation operator
 
-  int m_n_result; ///< the total number of result values
+  size_t m_n_result; ///< the total number of result values
```

Comment:
do we have to store this or can this be inferred from some other data member?

Reply:
I'll have a look

Reply:
replaced by a method in 59ec974